gpflow/multikernels.py

```diff
@@ -35,26 +50,38 @@ def Kdiag(self, X, presliced=False, full_cov_output=False):
         return tf.matrix_diag(stacked) if full_cov_output else stacked  # N x P x P  or  N x P
 
 
-class IndependentFeature(InducingPoints):
+class IndependentSharedInducingPoints(InducingPoints, IndependentFeature):
+    @params_as_tensors
     def Kuf(self, kern, Xnew):
-        pass  # N x P x M
+        if type(kern) is Independent:
```

Comment:
Yep I think we should make Kuf() and Kuu() multiple-dispatch functions instead of inducing features methods. Will do that in a separate PR to the multipledispatch_featureconditionals branch.